azure-pipelines.release-vnext-nightly.yml

```diff
@@ -80,6 +82,19 @@ jobs:
           artifactName: SBom-$(System.JobAttempt)
           targetPath: $(System.DefaultWorkingDirectory)/_manifest
 
+      # Since releases are scoped, this should warn for any packages that were mistakenly not included in scoping
+      - script: |
+          yarn syncpack list-mismatches
+        displayName: Check for dependency mismatches
+
+      - task: ComponentGovernanceComponentDetection@0
```

Comment:
Totally agree with this, we want to make sure partners can pick up nightlies if they need to access latest and greatest features 👍👍